CODE_OF_CONDUCT.md

```diff
@@ -1 +1 @@
-This project is governed by the Contributor Covenant version 1.4 (https://www.contributor-covenant.org/version/1/4/code-of-conduct.html). All contributors and participants agree to abide by its terms. To report violations, send an email to covenant@idolhands.com.
+This project is governed by the [Contributor Covenant version 1.4] (https://www.contributor-covenant.org/version/1/4/code-of-conduct.html). All contributors and participants agree to abide by its terms. To report violations, send an email to covenant@idolhands.com.
```

Comment:
The link appears to be broken on GitHub's markdown renderer. While I think the original Markdown implementation allowed a space between the `]` and `(` characters in a link, both GitHub Flavored Markdown and CommonMark seem to disallow it. Anyway, that was probably more verbiage than an extraneous space warranted, sorry.